2023-06-08 09:44:01.552 [info] Эксперимент «pythonPromptNewFormatterExt» активен
2023-06-08 09:44:01.552 [info] Эксперимент «pythonPromptNewToolsExt» активен
2023-06-08 09:44:01.552 [info] Эксперимент «pythonSurveyNotificationcf» активен
2023-06-08 09:44:01.552 [info] Test server listening.
2023-06-08 09:44:01.552 [info] > conda info --json
2023-06-08 09:44:01.581 [info] Found: /bin/python3 --> /bin/python3.10
2023-06-08 09:44:01.581 [info] Found: /bin/python3.10 --> /bin/python3.10
2023-06-08 09:44:01.615 [info] Found: /usr/bin/python3 --> /usr/bin/python3.10
2023-06-08 09:44:01.615 [info] Found: /usr/bin/python3.10 --> /usr/bin/python3.10
2023-06-08 09:44:01.642 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2023-06-08 09:44:01.642 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2023-06-08 09:44:01.793 [info] Found: /usr/local/bin/python --> /usr/local/bin/python3.11
2023-06-08 09:44:01.793 [info] Found: /usr/local/bin/python3 --> /usr/local/bin/python3.11
2023-06-08 09:44:01.793 [info] Found: /usr/local/bin/python3.11 --> /usr/local/bin/python3.11
2023-06-08 09:44:01.926 [info] > /bin/python3 -I ~/.vscode-remote/extensions/ms-python.python-2023.8.0/pythonFiles/get_output_via_markers.py ~/.vscode-remote/extensions/ms-python.python-2023.8.0/pythonFiles/interpreterInfo.py
2023-06-08 09:44:01.953 [info] > /usr/bin/python3 -I ~/.vscode-remote/extensions/ms-python.python-2023.8.0/pythonFiles/get_output_via_markers.py ~/.vscode-remote/extensions/ms-python.python-2023.8.0/pythonFiles/interpreterInfo.py
2023-06-08 09:44:02.744 [info] > /usr/local/bin/python -I ~/.vscode-remote/extensions/ms-python.python-2023.8.0/pythonFiles/get_output_via_markers.py ~/.vscode-remote/extensions/ms-python.python-2023.8.0/pythonFiles/interpreterInfo.py
2023-06-08 09:44:03.473 [info] Путь к интерпретатору Python: /usr/local/bin/python
2023-06-08 09:44:04.955 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2023-06-08 09:44:04.955 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2023-06-08 09:44:04.955 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2023-06-08 09:44:04.955 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2023-06-08 09:44:04.955 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2023-06-08 09:44:04.955 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2023-06-08 09:44:04.955 [warning] Retry failed, storage update failed for key PYTHON_WORKSPACE_STORAGE_KEYS
2023-06-08 09:44:04.955 [warning] Retry failed, storage update failed for key PYTHON_WORKSPACE_STORAGE_KEYS
2023-06-08 09:44:04.955 [warning] Retry failed, storage update failed for key PYTHON_WORKSPACE_STORAGE_KEYS
2023-06-08 09:44:11.233 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2023-06-08 09:44:11.233 [warning] Retry failed, storage update failed for key PYTHON_GLOBAL_STORAGE_KEYS
2023-06-08 09:44:11.855 [info] Запуск языкового сервера Pylance.
